Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -465,6 +465,7 @@ Renderer::Renderer()
 	g_Config.backend_info.bSupportsBBox = GLExtensions::Supports("GL_ARB_shader_storage_buffer_object");
 	g_Config.backend_info.bSupportsGSInstancing = GLExtensions::Supports("GL_ARB_gpu_shader5");
 	g_Config.backend_info.bSupportsGeometryShaders = GLExtensions::Version() >= 320;
+	g_Config.backend_info.bSupportsPaletteConversion = GLExtensions::Supports("GL_ARB_texture_buffer_object");
```

Comment:
The fact that this isn't supported unconditionally is going to be painful in terms of GameINI maintenance; we don't want to force on EFB2RAM in the GameINI for Twilight Princess just because we can't do palette conversion on  OpenGL ES.